modules/nested-tabs/assets/scss/frontend.scss

```diff
@@ -56,9 +52,6 @@
 		flex-direction: var( --n-tabs-direction );
 		gap: var( --n-tabs-gap );
 		text-align: $start;
-		height: var( --n-tabs-height );
-		overflow-x: var( --n-tabs-overflow-x );
-		overflow-y: var( --n-tabs-overflow-y );
```

Comment:
Remove unnecessary scrolling.
Originally we had implemented a `max-height` control for this widget, but we removed it during the development process.